cmd/solana_exporter/exporter.go

```diff
@@ -49,15 +51,17 @@ type SolanaCollector struct {
 	balances                *prometheus.Desc
 	isHealthy               *prometheus.Desc
 	numSlotsBehind          *prometheus.Desc
+	minimumLedgerSlot       *prometheus.Desc
```

Comment:
Nice! `minimumLedgerSlot`, `firstAvailableBlock` and `blockHeight` are all nice additions. 

Although, `blockHeight` is returned by the `GetEpochInfo` [endpoint](https://solana.com/docs/rpc/http/getepochinfo), so I would suggest that `blockHeight` be moved to the `SlotWatcher` and tracked easily in `slots.go`.

Reply:
moved to slots.go in GetEpochInfo.